modules/tfc-agent-mig-container-vm/main.tf

```diff
@@ -28,7 +28,7 @@ locals {
         path = "/var/run/docker.sock"
       }
   }] : []
-  network_name          = var.create_network ? google_compute_network.tfc_agent_network[0].self_link : var.network_name
+  network_name          = var.create_network ? google_compute_network.tfc_agent_network[0].self_link : try(var.network_name, null)
   subnet_name           = var.create_network ? google_compute_subnetwork.tfc_agent_subnetwork[0].self_link : var.subnet_name
```

Comment:
Just to check, can both `network_name` and `subnet_name` be defined at the same time? I thought not, and I ask because I see several situations in which this may happen (i.e.: `create_network` is `true`).

If it can be done, ignore this comment.

Reply:
I think yes, if create_network is true, the module creates a network first, and then a subnetwork inside the network just created (the module already has this behaviour, we are going to open a PR against Google's repo, so I think it is better to preserve this functionality. WDYT?